### DISCUSSION TITLE
HID not detected in Chrome OS

### DISCUSSION THREAD

**Discussion by amahajan-code** [NicoHood/HID#368] [Q&A]
Hello,

I am able to use Arduino Pro Micro as composite keyboard mouse device with this HID library. Arduino Pro Micro gets detected as composite HID device on Windows/Linux environment. Composite device works well with these host operating systems. However, if I connect to a PC with Chrome OS installed then HID device is not detected at all. Look forward for your suggestions. Thank you!

**Comment by NicoHood** [comment]
Does it work with the official arduino hid library?

**Comment by amahajan-code** [comment]
official arduino hid library ==> Are you referring to this ?
https://www.arduino.cc/reference/en/libraries/hid-project/

**Reply by NicoHood** [reply]
no, this: https://www.arduino.cc/reference/en/language/functions/usb/keyboard/

**Comment by amahajan-code** [comment]
No. I did not try this. Will it work on Pro Micro?

**Comment by amahajan-code** [comment]
BTW, I need both mouse and keyboard

**Reply by NicoHood** [reply]
then try it!!

**Comment by amahajan-code** [comment]
We need keyboard mouse in pre-boot / BIOS environment also.  That was the reason for not trying earlier. Does it have it now? I will try anyways.

Thanks for replying and helping. BTW, let me know if we have an option to work with your company for development work. It will be great to collaborate.

Thanks,
-Ayesh

**Reply by NicoHood** [reply]
As we want to debug the issue we need to know if it is in this library or the arduino stay. I would guess in the arduino stack, but you have to confirm by testing their library instead for now.

**Comment by amahajan-code** [comment]
Ok!
________________________________
